src/web/mjs/connectors/MangaTown.mjs

```diff
@@ -66,6 +66,7 @@ export default class MangaTown extends Connector {
     }
 
     _handleConnectorURI( payload ) {
+        this.requestOptions.headers.set("x-referer", "mangahere.com");
```

Comment:
This will add `MangaHere` to all request for this connector (manga lists, chapter lists, ...), is this intentional?
If not create a clone of request options and add the header to the clone.

Reply:
personally i'd rather reject the request and alert("use mangahere directly")

Reply:
Hum, you are right, better not meddle with the entire request options.

Reply:
In fact setting the referer to mangahere is only needed for downloading pictures, which is in fine handled by the parent class "handleconnectorURI". And this one use this.requestoptions without referer. No matter what i do in mangatown own handleconnectorURI its useless .

Is there any reason we are using handleconnectorURI in this connector? How could i set a referer but only for picture requests?

Reply:
Just set the header on the cloned request...
```javascript
let request = new Request(payload, this.requestOptions);
request.headers.set('x-referer', 'https://mangahere.com');
```

Reply:
This request get the page with the pics, and we dont need mangahere ref for that. :)

Downloading pictures need mangahere ref.

Which is done not in Mangatown.mjs but in parent class, where i cant pass a custom referrer .
Wait, can we pass a custom referer in payloads?

Reply:
I managed to fix the code so only picture requests get to use mangahere referer.